builder/osc/common/step_create_tags.go

```diff
@@ -88,31 +84,29 @@ func (s *StepCreateTags) Run(_ context.Context, state multistep.StateBag) multis
 			return multistep.ActionHalt
 		}
 		snapshotTags.Report(ui)
-
 		// Retry creating tags for about 2.5 minutes
 		err = retry.Run(0.2, 30, 11, func(_ uint) (bool, error) {
 			// Tag images and snapshots
-			_, _, err := regionconn.TagApi.CreateTags(context.Background(), &osc.CreateTagsOpts{
-				CreateTagsRequest: optional.NewInterface(osc.CreateTagsRequest{
-					ResourceIds: resourceIds,
-					Tags:        amiTags,
-				}),
-			})
+			request := oscgo.CreateTagsRequest{
+				ResourceIds: resourceIds,
+				Tags:        omiTags,
+			}
+
+			_, _, err := regionconn.Api.TagApi.CreateTags(regionconn.Auth).CreateTagsRequest(request).Execute()
 			if awsErr, ok := err.(awserr.Error); ok {
 				if awsErr.Code() == "InvalidOMIID.NotFound" ||
 					awsErr.Code() == "InvalidSnapshot.NotFound" {
 					return false, nil
 				}
 			}
```

Comment:
These will not work anymore, we could plan to update them

Reply:
I ll do it in a seperate PR

Reply:
Ok good